cat/CaT-service.yaml

```diff
@@ -4521,36 +4560,64 @@ components:
       description: 
         Details of an Evaluation type. This is known as an Envelope or Criterion
       type: object
-      properties:
-          id:
-            title: Identifier
-            description: 'The identifier for this criterion. It must be unique and cannot change within the Open Contracting Process it is part of (defined by a single ocid). See the [identifier guidance](http://standard.open-contracting.org/latest/en/schema/identifiers/) for further details.'
-            type: string
-            minLength: 1
-            readOnly: true
-          title:
-            title: Title
-            description: A title for this criterion.
-            nullable: true 
-            type: string
-            minLength: 1          
-          description:
-            title: Description
-            description: A description of this criterion. Structured information should be provided in its other fields.
-            nullable: true
-            type: string
-            minLength: 1 
-          relatesTo:
-            allOf:
-              - $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/OCDS/OCDS_CodeLists_Ext.yaml#/components/schemas/RelatesTo'
-          relatedItems:
-            type: string
-          requirementGroups:
-            type: array
-            items:
-              $ref: '#/components/schemas/QuestionGroup'
-            minItems: 1
+      allOf:
```

Comment:
`allOf` will flatten this property list. In other responses we have distinct `OCDS` / `NonOCDS` structures - do we want to make that consistent here? (will not be backwardly compatible with UI code, but will be consistent in the API). Just a thought?

Reply:
Yes I would like to do that but it will mean refactoring all of Prathap's code. I have agreed to flatten for MVP and refactor post MVP